<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nova consulta</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NOVA CONSULTA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Informações do paciente</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="9" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Plano de saúde</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLineEdit" name="fone_paciente">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nome</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Telefone p/ contato</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLineEdit" name="idade_paciente">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLineEdit" name="id_paciente">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QComboBox" name="combo_paciente">
             <item>
              <property name="text">
               <string>Selecione....</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Idade</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QComboBox" name="combo_plano">
             <item>
              <property name="text">
               <string>Sim</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Não</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Informações do médico </string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="0">
            <widget class="QLineEdit" name="id_medico">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Telefone p/ contato</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="combo_medico">
             <item>
              <property name="text">
               <string>Selecione....</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Nome</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLineEdit" name="fone_medico">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Área de atuação</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLineEdit" name="area_medico">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Consulta</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Tipo</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QTextEdit" name="obs"/>
           </item>
           <item row="8" column="0">
            <widget class="QPushButton" name="salvar_consulta">
             <property name="text">
              <string>Marcar consulta</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Observações adicionais</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLineEdit" name="valor">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QDateTimeEdit" name="data_consulta"/>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="combo_consulta">
             <item>
              <property name="text">
               <string>Selecione....</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1° Consulta</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Retorno</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Valor</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Data e hora</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Limpar dados</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Consultas marcadas</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Paciente</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Médico</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Área</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Data e hora</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_btn">
           <property name="text">
            <string>Cancelar consulta</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
